docs/usage/self-hosted-configuration.md

```diff
@@ -667,6 +667,50 @@ By default, Renovate does not autodiscover repositories that are mirrors.
 
 Change this setting to `true` to include repositories that are mirrors as Renovate targets.
 
+## inheritConfig
+
+When you enable this option, Renovate will look for the `inheritConfigFileName` file (defaults to `org-inherited-config.json`) in the `inheritConfigRepoName` repository (defaults to `{{parentOrg}}/renovate-config`) before processing a repository, and read this in as config.
+If the repository is in a nested organization/group on a supported platform such as GitLab, such as `topGroup/nestedGroup/projectName` then Renovate will look in `topGroup/nestedGroup/renovate-config`.
```

Comment:
```suggestion
When you enable this option, Renovate will look for the `inheritConfigFileName` file in the `inheritConfigRepoName` repository before processing a repository, and read this in as config.

| Config option           | Default path                | Repository config       | Repository path                 |
| ----------------------- | --------------------------- | ----------------------- | ------------------------------- |
| `inheritConfigFileName` | `org-inherited-config.json` | `inheritConfigRepoName` | `{{parentOrg}}/renovate-config` |

If the repository is in a nested organization or group on a supported platform such as GitLab, such as `topGroup/nestedGroup/projectName` then Renovate will look in `topGroup/nestedGroup/renovate-config`.
```

We should move the "defaults to `x`" info out of the introductory sentence, because that makes the sentence very hard to read. Maybe a table like this will work for that? Or you could split the default setting into their own sentences or list.